tools/mtf

```diff
@@ -13,8 +15,39 @@ positional arguments:
   AVOCADO_ARGS        test files and any arguments that "avocado run" accepts
 
 optional arguments:
+  -a, --action        run selected avocado command (DEFAULT: run)
   -h, --help          show this help message and exit
   -l, --linters       also run default linter tests included in MTF
+  -t, --tags T1,-T2   set test filtering based on tags, use --taglist for help ()
+  --taglist           oficially supported tag list (you can use whatever you want, this is list oficially used)
+                      for more info look at:
+                      http://avocado-framework.readthedocs.io/en/55.0/Loaders.html#filtering-tests-by-tags
+
+EOF
+}
+
+function print_tags {
+  cat <<EOF
+list of well known tags:
+  test environments:
+    rhel     - working on RedHat Enterprise Linux and CentOs
+    fedora   - working on Fedora
+    docker   - working on Docker
+
+  test categorization:
+    tier1    - basic qualification tests
```

Comment:
I would name it differently. tier1 does not sound well. My opinion.

Reply:
Perhaps "sanity" would be a good name?

Reply:
I'm not against, but there is more troubles with __sanity__
there is more levels of testing: like unittest, component tests (smoke, sanity, regressions), integrations tests, compose testing and much more. 
these levels and sometimes this can cause misunderstanding on both sides :-)
So ther is possible have tags like: `tier0, tier1, tier2, tier3` ... or have tags like `unittest, smoke, sanity, regression, integration, ...` 
* __tierX__ - not descriptive, but easy to understand that there is more levels, and that levels has some structure
* __sanity,... - descriptive, but not easy to see structure of levels

I'm not against any solution. or maybe we can tag these tests by both tags, `tier1` and `sanity`, it allows to use both names